images/docs-builder/Dockerfile

```diff
@@ -7,8 +7,7 @@ RUN yum update -y && \
     yum module install nodejs:14 -y && \
     yum clean all && rm -rf /var/cache/yum/*
 
-RUN gem install asciidoctor asciidoctor-pdf --pre && \
-    npm install -g asciidoc-link-check
+RUN npm install -g asciidoc-link-check
```

Comment:
@robin-owen `asciidoctor` is required to build the docs, we just need to remove the `asciidoctor-pdf`